### PR TITLE
Added download button, add button now adds blank culture

### PR DESCRIPTION
**Pull request by nlamanna18**
Story: [88](https://5911-capstone.atlassian.net/browse/OCA-88)

## Summary
Home page now has download button next to each culture that has functionality. Add culture button appears only when user is logged in and adds blank culture.

### CODE REVIEW

File: frontend/views/Home.tsx [hunk 1]
```diff
@@ -139,19 +129,38 @@ function Home(props: Props) {
...
+                      icon="delete"
+                      onPress={() => Culture.delete(item.name, props.token)}
+                    />
+                  )}
+                </View>
+              )}
             />
           );
         }}
         //ListEmptyComponent={EmptyListMessage}
```
Comment (freeman91) [inline frontend/views/Home.tsx:155]:
```suggestion
```

**Review by freeman91 — CHANGES_REQUESTED** [review]
Just an FYI typically you should create a new branch when starting a new piece of work. Nothing wrong with using React-Nav but it could be confusing

**Comment by NickHackman** [comment]
@nlamanna18 yes, please delete React-Nav this branch is old. Please make future changes off of new branches starting at `origin/master`

**Review by nlamanna18 — COMMENTED** [review]
Sounds good.

**Review by freeman91 — APPROVED** [review]
Looks Good. I resolved your merge conflicts so you should be good to merge

**Pull request merged**